src/Overcrowdin/Overcrowdin.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There's no need to add a .nusepc file when using the sdky-style .csproj files. `msbuild /t:pack` (or the corresponding `dotnet` command) will use the information from the .csproj file to create the nuget package.